trace/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "trace"
```

Comment:
I suspect you want a name like `datadog-trace-normalization` and to name the folder `trace-normalization`. At least with current conventions, which we can definitely change as we grow if we need to.

Reply:
makes sense, I'll change that!